.github/actions/job-preamble/action.yaml

```diff
@@ -27,9 +27,9 @@ runs:
           /opt/hostedtoolcache/Ruby
         )
         for d in "${cleanup[@]}"; do
-          if [[ -d "/mnt${d}" ]]; then
+          if [[ -e "/mnt${d}" ]]; then
             rm -rf "/mnt${d}"
-          elif [[ -d "$d" ]]; then
+          elif [[ -e "$d" ]]; then
             rm -rf "$d" \
               || sudo rm -rf "$d"
```

Comment:
@jvdm also pointed out we could do something like this instead:
```
          rm -rf "/mnt${d}" \
              || sudo rm -rf "/mnt${d}"
          rm -rf "$d" \
              || sudo rm -rf "$d"
```
I don't have a strong preference.

Reply:
I think the second is cleaner from a bash-syntax perspective. But, notice the second always removes "$d", while the first only removes it if "/mnt$d" does not exist.